WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipment Tracking Section/Add Tracking/ManualTrackingViewController.swift

```diff
@@ -228,16 +228,17 @@ extension ManualTrackingViewController: UITableViewDataSource {
     }
 
     private func configureShippingProvider(cell: TitleAndEditableValueTableViewCell) {
-        cell.title.text = NSLocalizedString("Shipping provider", comment: "Add / Edit shipping provider. Title of cell presenting name")
+        cell.title.text = NSLocalizedString("Shipping carrier", comment: "Add / Edit shipping carrier. Title of cell presenting name")
         cell.value.text = viewModel.providerCellName
+        cell.value.placeholder = NSLocalizedString("Select carrier", comment: "Add the shipping carrier. Placeholder of cell presenting carrier name")
```

Comment:
this seems to be also implemented in https://github.com/woocommerce/woocommerce-ios/pull/1948 - not sure if you'd like to close https://github.com/woocommerce/woocommerce-ios/pull/1948 in favor of this PR or remove this line from this PR?

Reply:
I'll merge only this PR, and I'll close the first one #1948.